src/msg/Messenger.cc

```diff
@@ -207,7 +207,7 @@ bool Messenger::ms_deliver_verify_authorizer(
 	connection_secret,
 	challenge);
       if (isvalid) {
-	dis->ms_handle_authentication(con);
+	return dis->ms_handle_authentication(con) < 0;
```

Comment:
I think what we want is to return `dis->ms_handle_authentication(con) == 0` or `dis->ms_handle_authentication(con) >= 0`

Reply:
Ah!! Oops! That would explain why my vstart cluster is broken.